standalone/inc/pup/PUPTriggerCondition.cpp

```diff
@@ -0,0 +1,40 @@
+#include "core/stdafx.h"
+
+#include "PUPTriggerCondition.h"
+
+#include "PUPManager.h"
+
+PUPTriggerCondition::PUPTriggerCondition(const string& szId, int value)
```

Comment:
Where did the error handling go? If I write a pup pack with invalid triggers I want to know.

Reply:
Sorry if I missed something. Can you add some
more details and I’ll put it back.

Reply:
I'd say this would never happen, but we have seen some crazy things in pup files.

When I refactored, instead of `stoi`, I used all the helper functions we made along the way and are all over in pup and flex. In this case specifically `string_to_int`, (which leverages more custom functions `try_parse_int` which leverages `trim_string`)